mcs/swagger.yml

```diff
@@ -8,6 +8,14 @@ produces:
   - application/json
 schemes:
   - http
+securityDefinitions:
+  key:
+    type: oauth2
+    flow: accessCode
+    authorizationUrl: http://min.io
+    tokenUrl: http://min.io
+security:
+  - key: []
```

Comment:
can you please add comments `#...` next to this definitions to add more doc of what they do

Reply:
sure